pkg/util/detector/detector.go

```diff
@@ -38,6 +40,17 @@ import (
 	"github.com/karmada-io/karmada/pkg/util/restmapper"
 )
 
+const (
+	// DeploymentKind indicates the target resource is a deployment
+	DeploymentKind = "Deployment"
```

Comment:
We have the same constants here https://github.com/karmada-io/karmada/blob/master/pkg/util/constants.go#L70. I suggest to put them at the same place.

Reply:
ok，I will modify it.